src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBaseKernels.cpp

```diff
@@ -17,302 +17,11 @@
-                      arrayView4d< real64 const, constitutive::multifluid::USD_PHASE_DC > const & dPhaseDens_dComp, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & phaseFrac, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & dPhaseFrac_dPres, \
-                      arrayView4d< real64 const, constitutive::multifluid::USD_PHASE_DC > const & dPhaseFrac_dComp, \
-                      arrayView2d< real64, compflow::USD_PHASE > const & phaseVolFrac, \
-                      arrayView2d< real64, compflow::USD_PHASE > const & dPhaseVolFrac_dPres, \
-                      arrayView3d< real64, compflow::USD_PHASE_DC > const & dPhaseVolFrac_dComp ); \
-  template \
-  void \
-  PhaseVolumeFractionKernel:: \
-    launch< NC, NP >( SortedArrayView< localIndex const > const & targetSet, \
-                      arrayView2d< real64 const, compflow::USD_COMP > const & compDens, \
-                      arrayView2d< real64 const, compflow::USD_COMP > const & dCompDens, \
-                      arrayView3d< real64 const, compflow::USD_COMP_DC > const & dCompFrac_dCompDens, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & phaseDens, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & dPhaseDens_dPres, \
-                      arrayView4d< real64 const, constitutive::multifluid::USD_PHASE_DC > const & dPhaseDens_dComp, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & phaseFrac, \
-                      arrayView3d< real64 const, constitutive::multifluid::USD_PHASE > const & dPhaseFrac_dPres, \
-                      arrayView4d< real64 const, constitutive::multifluid::USD_PHASE_DC > const & dPhaseFrac_dComp, \
-                      arrayView2d< real64, compflow::USD_PHASE > const & phaseVolFrac, \
-                      arrayView2d< real64, compflow::USD_PHASE > const & dPhaseVolFrac_dPres, \
-                      arrayView3d< real64, compflow::USD_PHASE_DC > const & dPhaseVolFrac_dComp )
-
-INST_PhaseVolumeFractionKernel( 1, 1 );
-INST_PhaseVolumeFractionKernel( 2, 1 );
-INST_PhaseVolumeFractionKernel( 3, 1 );
-INST_PhaseVolumeFractionKernel( 4, 1 );
-INST_PhaseVolumeFractionKernel( 5, 1 );
-
-INST_PhaseVolumeFractionKernel( 1, 2 );
-INST_PhaseVolumeFractionKernel( 2, 2 );
-INST_PhaseVolumeFractionKernel( 3, 2 );
-INST_PhaseVolumeFractionKernel( 4, 2 );
-INST_PhaseVolumeFractionKernel( 5, 2 );
-
-INST_PhaseVolumeFractionKernel( 1, 3 );
-INST_PhaseVolumeFractionKernel( 2, 3 );
-INST_PhaseVolumeFractionKernel( 3, 3 );
-INST_PhaseVolumeFractionKernel( 4, 3 );
-INST_PhaseVolumeFractionKernel( 5, 3 );
-
-#undef INST_PhaseVolumeFractionKernel
-
-} // namespace CompositionalMultiphaseBaseKernels
+{} // namespace CompositionalMultiphaseBaseKernels
```

Comment:
My guess would be we're not going back to explicit instantiations, so let's remove the empty files(s)?

Reply:
I removed two empty files in the last commit: `SinglePhaseBaseKernels.cpp` and `CompositionalMultiphaseBaseKernels.cpp`